src/main/java/utilities/ShellUtility.java

```diff
@@ -349,28 +358,39 @@ public static Response executeCommand(Command command, TypeOfShell typeOfShell)
      * Executes the specified command
      *
      * @param command         Command to execute
-     * @param timeOutDuration Time out for the cmmand execution
+     * @param timeOutDuration Time out for the command execution
```

Comment:
Do we need to specify if the timeout is in seconds/milliseconds?

Reply:
Not needed as the unit will be specified inside the Duration object.